docs/lib/graphqlapi/fragments/flutter/authz/10_userpool.md

```diff
@@ -0,0 +1,8 @@
+Add the following code to your app:
+
+```dart
+await amplify.addPlugin(
+    authPlugins: [AmplifyAuthCognito()]
+    apiPlugins: [AmplifyAPI()]
+);
```

Comment:
Needs to be updated as per the refactor.

Reply:
changed to: 

```dart
await amplify.addPlugins([
    AmplifyAuthCognito(),
    AmplifyAPI()
]);
```

Reply:
Why should not update per the refactor in this PR, but in the PR for the refactor. This will go out before the release